### PR TITLE
Option to prune actionwords

### PR DESCRIPTION
**Pull request by mhfrantz**
Tests cover both with and without test runs.  The difference is that with test runs, the XML will include only snapshots, while without specifying a test run, the XML includes only the base objects (scenarios, actionwords).

### CODE REVIEW

File: spec/hiptest_publisher_spec.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def have_not_printed(message)
 
   after(:each) {
     if @output_dir_created
-      FileUtils.rm_rf(output_dir)
+      # FileUtils.rm_rf(output_dir)
```
Comment (duane-talentsonar) [inline spec/hiptest_publisher_spec.rb:29]: Has this commented out code been left for a reason?

**Comment by duane-talentsonar** [comment]
FTP

File: spec/hiptest_publisher_spec.rb (hunk 1 above)
Comment (mhfrantz) [inline spec/hiptest_publisher_spec.rb:29]: No, that was a mistake.  Thanks for noticing.

**Comment by mhfrantz** [comment]
Replaced by #10

**Pull request closed without merging**